src/components/TablePreview/TablePreview.tsx

```diff
@@ -56,7 +56,9 @@ const TablePreview: FC<Props> = ({}) => {
                                 <tr>
                                     <td style={title.style}>{title.textValue}</td>
                                     <td style={spacerStyle}></td>
-                                    <td style={email.style}>{email.textValue} </td>
+                                    <td style={email.style}>
+                                        {email.textValue === undefined ? <></> : email.textValue}{" "}
+                                    </td>
```

Comment:
We shouldn't need this anymore since we're only updating the state when the input is valid